root/documentation/gabeacon.conf

```diff
@@ -40,34 +40,6 @@
         example=__VAR(GA4GH_beacon_start)__
         required=1
       </start>
```

Comment:
Documentation for `start` and `end` parameters should mention the support of genomic ranges using two comma-separated values.

Reply:
Good point. Updated.